README.md

```diff
@@ -25,3 +25,15 @@ To run the tests:
 ```bash
 npm test
 ```
+
+## Test app
+
+This is very much a work in progress.
+
+To run the test app:
+
+```
+$ cd test/app
```

Comment:
I don't really feel strongly about it, but I think `test/` should be reserved for automated tests. Maybe we could put it in a new folder called `app/` or something?